### ISSUE TITLE
GWT module iframe does not have a title attribute for accessibility

### ISSUE THREAD

**Issue by dvlpr-o** [gwtproject/gwt#9443]
Adding a GWT module script to a page creates an `iframe` that does not have a `title` attribute. Accessibility guidelines indicate that `frame` and `iframe` elements should contain a `title` attribute to describe the contents of each frame. This provides a label for the frame so users can determine which frame to enter and explore in detail.

[WCAG 2.0](https://www.w3.org/TR/WCAG20-TECHS/H64.html)
[Quick Reference Guide to Section 508 Requirements and Standards](https://section508.gov/content/quick-reference-guide#1194.22i)

**Comment by jnehlmeier** [comment]
I guess the iframe should be hidden from screen readers, as it is an implementation detail of GWT and only contains JS code. There is no HTML to explore in that iframe.

**Comment by dvlpr-o** [comment]
Would the browser load the `script` element within the `iframe` if the `iframe` is hidden?

**Comment by tbroyer** [comment]
I see that Google Groups has a hidden iframe for another script that reads:
```html
<iframe src="https://clients5.google.com/pagead/drt/dn/" aria-hidden="true" style="display: none !important;" hidden></iframe>
```
so I suppose GWT's iframe could be changed.

Anyone wants to experiment with this and report on the findings in many different browsers?

1. copy `installLocationIframe.js` and make the changes
2. create a new linker that `extends CrossSiteIframeLinker` and overrides `getJsInstallLocation` to return the patched script
3. finally `<define-linker>` and `<add-linker>`

**Comment by hannahiss** [comment]
Yes, this iframe should be hidden from screen readers with aria-hidden attribute, but I haven't been able to set it. Is there a way for it ?

**Comment by gtritchie** [comment]
I submitted a patch for this: https://gwt-review.googlesource.com/c/gwt/+/21860
